crescita-personale/benefit.md

```diff
@@ -1,32 +1,40 @@
 # Benefit
 
-Chiunque all'interno del nostro team ha diritto ai seguenti benefici.
-
-## Hardware e software
-
-Hardware e licenze software a scelta, qualunque cosa vi serva per svolgere al meglio il vostro 
-lavoro. Se hai bisogno di qualcosa, manda un messaggio su Slack a uno dei soci.
-
-## Settimane Nebulose
-
-Le Settimane Nebulose sono 2/3 meeting annuali di durata settimanale in cui si lavora tutti insieme.
-  
-## Flessibilità
-
-Tutti nel team hanno diritto a lavorare in orari flessibili (compatibilmente con le esigenze dei
-clienti e degli altri membri del team) e a lavorare da remoto.
-
-## Conferenze
-
-Le spese per partecipare a eventuali conferenze (biglietto aereo, hotel e biglietto per la
-conferenza) sono completamente coperte da Nebulab. 
-
-## Istruzione
-
-Tutti i membri del team possono avere accesso a libri, e-book e corsi online.
-
-Abbiamo account di gruppo su vari siti che offrono corsi online e altre risorse. Per le credenziali,
-chiedere a uno dei soci.
-
-In caso di necessità è possibile richiedere l'acquisto di libri o servizi aggiuntivi di cui si ha 
-bisogno.
+Tutti i membri del team hanno diritto ai benefit elencati in questo documento. Se avete dubbi su uno
+dei benefit, non siate timidi e chiedete! 
+
+- **Hire kit.** Tutti i membri del team ricevono un kit hardware all'assunzione composto da:
+  - [MacBook Pro da 13"](https://www.apple.com/it/shop/buy-mac/macbook-pro/13%22-grigio-siderale-dual%E2%80%91core-a-2,3ghz-128gb#) (16 GB di RAM, SSD da 256 GB, tastiera US o IT);
+  - [sleeve Tucano Second Skin Elements](https://www.amazon.it/dp/B009F8ARNS/ref=pe_3310731_189395851_TE_dp_1); 
+  - [Apple Magic Keyboard](https://www.apple.com/it/shop/product/MLA22T/A/magic-keyboard-italiano) (con layout a scelta tra US e IT);
+  - mouse a scelta tra [Apple Magic Mouse](https://www.apple.com/it/shop/product/MRME2Z/A/magic-mouse-2-grigio-siderale) e [Logitech MX Master 2S](https://www.logitech.com/it-it/product/mx-master-2s-flow);
+  - cuffie a cancellazione attiva del rumore (a scelta tra [Bose QuietComfort 35 II](https://www.bose.it/it_it/products/headphones/over_ear_headphones/quietcomfort-35-wireless-ii.html) e [Sony WH-1000XM3](https://www.sony.it/electronics/cuffie-archetto/wh-1000xm3));
+  - [monitor LG 27UK850-W](https://www.lg.com/it/monitor/lg-27UK850-W) (IPS, 27", 4K);
+  - [scrivania BEKANT](https://www.ikea.com/it/it/catalog/products/S69282592/#/S19022808) (delle dimensioni desiderate) e [sedia da ufficio JÄRVFJÄLLET](https://www.ikea.com/it/it/catalog/products/S99275632/#/S89275623).
+- **[Mentoring](https://github.com/nebulab/playbook/blob/master/crescita-personale/mentoring-e-1-1.md).**
+  Quando entri a far parte di Nebulab, ti sarà assegnato un mentore che è il tuo punto di 
+  riferimento non solo per tutto quello che riguarda l'azienda, ma anche per la tua crescita 
+  personale e professionale. Se sei interessato a un argomento e vorresti approfondirlo, ricorda che
+  il tuo mentore è lì per questo!
```

Comment:
Proposta:

"Quando entri a far parte del nostro team, ti viene assegnato un mentore, che sarà un punto di riferimento e sostegno durante tutto il tuo percorso all'interno di Nebulab, soprattutto per quel che riguarda la tua crescita personale e professionale. Ad esempio, se sei interessato a un argomento e vuoi approfondirlo, oppure sei in difficoltà su un progetto, rivolgiti senza indugio al tuo mentore, sarà felice di aiutarti!"